packages/react-components/src/components/FileUploadCards.tsx

```diff
@@ -34,6 +34,16 @@ export const _FileUploadCards = (props: FileUploadCardsProps): JSX.Element => {
   if (!files || files.length === 0) {
     return <></>;
   }
+  const iconButtonClassName = mergeStyles({
```

Comment:
Can we re-use the `iconButtonClassName` we created in `FileDownloadCards` component by moving them to a styles file?

Reply:
Done